test/utils.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+#
+# Utils for test scripts.
+#
+source $(dirname $BASH_SOURCE)/init_tools.sh
+
+log() {
+    echo `date +%Y/%m/%d:%H:%M:%S`" $@"
+}
+export -f log
+
+generate_test_id() {
+    echo $(cat /dev/urandom | tr -dc 'a-z' | fold -w 8 | head -n 1)
+}
+
+clean_resource_by_id() {
+    test_id=$1
+    KUBECTL="kubectl --namespace default"
+    set +e
+
+    crds=$($KUBECTL get crd | grep "fission.io" | awk '{print $1}')
+    crds="$crds configmaps secrets"
```

Comment:
Also add `ingress` - we do create ingress in some test I think

Reply:
Never mind - you are deleting route later and I think it will take care of this

Reply:
`ingress`, which is created by fission, will be deleted later by deleting `*.fission.io` crds or`fission route delete`.
`configmaps` and `secrets` are the only resources that will be created "manually" by kubectl.